packages/contracts/scripts/shared/helpers/flow.ts

```diff
@@ -0,0 +1,10 @@
+export const pressAnyKey = async (msg = "Press any key to continue"): Promise<void> => {
+    return new Promise((resolve) => {
+        console.log(msg || "Press any key to continue");
+        process.stdin.setRawMode(true);
+        process.stdin.resume();
+        process.stdin.on("data", () => {
+            resolve(undefined);
```

Comment:
We probably should have allowed for a way to cancel (`^c` just continues)

Reply:
This is a good catch! 

fixed in https://github.com/ubiquity/ubiquity-dollar/pull/261/commits/3de07c5d0c4fb0fd4c15b16664579f80373b9248